.github/workflows/build_and_test.yml

```
@@ -35,11 +35,11 @@ jobs:
   clippy:
     runs-on: ubuntu-latest
     steps:
-      - name: Install latest nightly
+      - name: Install stable toolchain
         uses: actions-rs/toolchain@v1
```

actionlint (1.7.12)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)